scripts/Dockerfile

```diff
@@ -16,4 +16,18 @@ ENV LANGUAGE en_US.UTF-8
 
 ENV PASH_TOP=/pash
 ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/"
+
+# NVM: NodeJS 8.9.4 is reportedly the lowest version used by stakeholders.
+ENV NVM_DIR /root/.nvm
+ENV NODE_VERSION 8.9.4
+RUN curl https://raw.githubusercontent.com/creationix/nvm/v0.37.2/install.sh | bash \
+    && . $NVM_DIR/nvm.sh \
+    && nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/versions/node/v$NODE_VERSION/lib/node_modules
+ENV PATH      $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
+RUN cd /pash/scripts/remote/controller && npm install && cd -
```

Comment:
Would it make sense to have those in a separate Dockerfile (only for the CI and controller)?

Reply:
If we are going for a separate repo for CI, then yes.

Reply:
My only hesitation is that the second Dockerfile would copy most of the first anyway.

Reply:
Hmm, I see. I guess I was thinking that the controller for the perf experiments is not included in the default environment for the project (a user can still run the perf experiments on their own machine without having node).

If however `node` does not increase docker build time I guess that we could leave it in as is.

Reply:
This exists too: https://docs.docker.com/develop/develop-images/multistage-build/